src/pages/workspaces/List.js

```diff
@@ -282,15 +273,61 @@ export const WorkspaceList = () => {
                   side: 'left'
                 }, [
                   Utils.switchCase(lastRunStatus,
-                    ['success', () => icon('success-standard', { size: 20, style: { color: colors.success() } })],
-                    ['failure', () => icon('error-standard', { size: 20, style: { color: colors.danger(0.85) } })],
-                    ['running', () => icon('sync', { size: 20, style: { color: colors.success() } })]
+                    ['success', () => icon('success-standard', {
+                      size: 20, style: { color: colors.success() },
+                      'aria-label': 'Workflow Status Success'
+                    })],
+                    ['failure', () => icon('error-standard', {
+                      size: 20, style: { color: colors.danger(0.85) },
+                      'aria-label': 'Workflow Status Failure'
+                    })],
+                    ['running', () => icon('sync', {
+                      size: 20, style: { color: colors.success() },
+                      'aria-label': 'Workflow Status Running'
+                    })]
```

Comment:
Let's add some screenreader text for the case when there is no submission icon to display so that you don't get "Blank" when you navigate to that column. Suggestion--

```
!!lastRunStatus ? h(TooltipTrigger, {
                  content: span(['Last submitted workflow status: ', span({ style: { fontWeight: 600 } }, [_.startCase(lastRunStatus)])]),
                  side: 'left'
                }, [
                  Utils.switchCase(lastRunStatus,
                    ['success', () => icon('success-standard', {
                      size: 20, style: { color: colors.success() },
                      'aria-label': 'Workflow Status Success'
                    })],
                    ['failure', () => icon('error-standard', {
                      size: 20, style: { color: colors.danger(0.85) },
                      'aria-label': 'Workflow Status Failure'
                    })],
                    ['running', () => icon('sync', {
                      size: 20, style: { color: colors.success() },
                      'aria-label': 'Workflow Status Running'
                    })]
                  )
                ]) : div({ className: 'sr-only' }, ['No workflows have been run'])